### PR TITLE
Update issue template for ReactNative and Flutter.

### PR DESCRIPTION
**Pull request by dnkoutso**

### CODE REVIEW

**Comment by orta** [comment]
"React Native" with the space 👍🏻

**Comment by dnkoutso** [comment]
fixing!

**Comment by dnkoutso** [comment]
fixed

**Review by orta — APPROVED** [review]

**Comment by dnkoutso** [comment]
fixed one more missing space.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+<!--
+WARNING: Please do not file issues for CocoaPods with projects that use React Native or Flutter. Please remove React Native or Flutter in your sample app in order
+to demonstrate properly an issue with CocoaPods itself. The integration between CocoaPods and React Native or Flutter makes it very difficult for us to diagnose
+and fix issues. Issues with React Native or Flutter will be closed.
```
Comment (paulb777) [inline .github/ISSUE_TEMPLATE.md:4]: May want to add a suggestion to file the issues with those project, perhaps with links?

**Review by paulb777 — APPROVED** [review]
Great idea!

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (dnkoutso) [reply .github/ISSUE_TEMPLATE.md:4]: yes!

**Pull request merged**